app/templates/components/task-item.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="card">
+  <header class="card-header">
+    <p class="card-header-title">
+      {{task.title}}
+    </p>
+    <a href="{{task.url}}" target="_blank" class="card-header-icon" aria-label="more options">
+      <span class="icon">
+        {{#if task.isPullRequest}}
+          {{fa-icon 'code-branch'}}
+        {{else}}
+          {{fa-icon 'bug'}}
+        {{/if}}
+      </span>
+    </a>
+  </header>
+  <div class="card-content">
+    <div class="content">
+      {{truncate-text task.bodyText limit=350}}
+    </div>
+    <time title="{{task.updatedAt}}">{{time-ago task.updatedAt}}</time>
+  </div>
+  <footer class="card-footer">
+    <a href="{{task.repository.url}}" class="card-footer-item" target="_blank">
+      <figure class="org-icon">
+        <img src="{{task.repository.owner.avatarUrl}}" alt="">
+      </figure>
+      &nbsp;
+      {{task.repository.nameWithOwner}}
+    </a>
+    <a href="{{task.author.url}}" class="card-footer-item" target="_blank">
```

Comment:
It isnt very clear that this is the author, especially for issues with an assignee.

Maybe, and im not very convinced of this, add a icon to indicate that it is an author?

This isnt very important because the user will quickly learn what this field is, so it wont hurt if it is a bit vague the first time they use it.

Reply:
I didn't think of it.  Not sure how an icon will help to differentiate author vs assignees. But my proposed solution would be just name it. `Author:  bekicot`.

I can't find any icon that differentiates between author and assignees.  This project use `fort-awesome`

Reply:
I think stay with your current UI for now.  We can think about this later.